apps/web/src/hooks/Tokens.ts

```diff
@@ -199,7 +199,7 @@ export function useCurrency(currencyId: string | undefined): Currency | ERC20Tok
   return isNative ? native : token
 }
 
-export function useOnRampCurrency(currencyId: string | undefined): NativeCurrency | OnRampCurrency | null | undefined {
+export function useOnRampCurrency(currencyId: string | undefined): NativeCurrency | Currency | null | undefined {
```

Comment:
just a tip to use question mark if a function parameter is optional

Reply:
when i wrote this function i was following the format of the useCurrency function defined above is which uses same notation. `
export function useCurrency(currencyId: string | undefined): Currency | ERC20Token | null | undefined `
the function param isnt optional. i believe the undefined is specified as the currencyId may may be undefined, especially for a second after page load

Reply:
actually `function useCurrency(currencyId: string | undefined)` equals `function useCurrency(currencyId?: string)` if we don't have multiple parameters